scripts/smoke-test.js

```diff
@@ -0,0 +1,66 @@
+const os = require('os');
+const fs = require('fs-extra');
+const path = require('path');
+const glob = require('glob');
+
+const packageData = require('../package');
+const run = require('../cli/run');
+
+
+function replacePlaceholders(content) {
+  return content
+    .replace(/dredd-hooks-\{\{mylanguage\}\}/g, 'dredd-hooks-python')
+    .replace(/import hooks/g, "import dredd_hooks as hooks")
+    .replace(/\.\{\{myextension\}\}/g, '.py');
+}
+
+function uncommentPythonCodeBlocks(content) {
+  let insidePythonCodeBlock = false;
+  let quotationMarksSeen = 0;
+
+  return content
+    .split(/\r?\n/)
+    .map((line) => {
+      if (insidePythonCodeBlock) {
+        line = line.replace(/^      \#/, '      ');
+        quotationMarksSeen += /^      """/.test(line) ? 1 : 0;
+        insidePythonCodeBlock = quotationMarksSeen < 2;
+      } else {
+        quotationMarksSeen = 0;
+        insidePythonCodeBlock = /(Given|And) a file named ".+\.py"/i.test(line);
+      };
+      return line;
+    })
+    .join('\n');
+}
+
+
+// create a temporary directory and init an npm package there
+const testDir = fs.mkdtempSync(path.join(os.tmpdir(), 'dredd-hooks-template-test-'));
+run('npm', ['init', '--yes'], { cwd: testDir });
```

Comment:
This test illustrates nicely how this package is meant to be used. Maybe let's also suggest `npm init --yes` instead of a few manual steps (create `package.json`, fill it with an example ,etc.)?

Reply:
I had it like that, but then the `npm init` creates a lot of stuff in the file which is unnecessary unless one is creating a true npm package. And I think it's the best to have `private: true` there in this particular case, to prevent accidental publish or something. So in the end I decided to recommend creating the file manually, in as minimal manner as possible.

Reply:
Sounds reasonable. I don't see any harm in recommending the approach you've mentioned in the README.